common/views/components/Modal/Modal.tsx

```diff
@@ -142,19 +157,88 @@ const ModalWindow = styled(Space).attrs({
   }
 `;
 
-const Modal = ({
+const ModalWindowPaddingNoOverflow = styled(Space).attrs<BaseModalProps>({
```

Comment:
Could this be something like `ModalWindowPaddingNoOverflow = styled(BaseModalWindow)` so you don't have to repeat lots of the styling?

Reply:
Make alot of sense. I thought you would not be able to override the padding-left and padding right but did this.